src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -1141,6 +1143,7 @@ void runtime::cpu::CPU_ExternalFunction::build()
     pass_manager.register_pass<ngraph::pass::CommonSubexpressionElimination>();
     pass_manager.register_pass<ngraph::pass::CoreFusion>();
     pass_manager.register_pass<runtime::cpu::pass::CPUFusion>();
+    pass_manager.register_pass<runtime::cpu::pass::CPUInceptionHorizontalFusion>();
```

Comment:
do we need a separate pass? I would prefer if this is added to cpu_fusion. Also we dont need to say Inception here. This is more generic than the inception model.